src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/MethodCallExpression.cs

```diff
@@ -22,7 +22,7 @@ internal MethodCallExpression(MethodInfo method)
             Method = method;
         }
 
-        internal virtual Expression GetInstance() => null;
+        internal virtual Expression GetInstance() => null!;
```

Comment:
>Expression [](start = 25, length = 10)

It looks like there a number of derived classes that do not override this method. Given that, should this be declared as `Expression?` instead?

Reply:
Right, will update